libs/@guardian/source-react-components/src/checkbox/theme.ts

```diff
@@ -34,3 +47,27 @@ export const checkboxThemeBrand = {
 	...userFeedbackThemeBrand,
 	...labelThemeBrand,
 };
+
+export const checkboxTheme: CheckboxTheme = {
+	borderUnselected: palette.neutral[46],
+	borderHover: palette.brand[500],
+	borderSelected: palette.brand[500],
+	borderError: palette.error[400],
+	backgroundTick: palette.brand[500],
+	backgroundUnselected: 'transparent',
+	textLabel: palette.neutral[7],
+	textLabelSupporting: palette.neutral[46],
```

Comment:
Similarly to above, the Radio button theme is using `textSupporting` to match Figma.